gnosis/eth/tests/eip712/test_eip712.py

```diff
@@ -4,6 +4,43 @@
 
 
 class TestEIP712(TestCase):
+    address = "0x8e12f01dae5fe7f1122dc42f2cb084f2f9e8aa03"
```

Comment:
We should use checksummed addresses unless we are testing not checksummed addresses:

`0x8e12f01DAE5FE7f1122Dc42f2cB084F2f9E8aA03`